src/petals/client/sequential_autograd.py

```diff
@@ -141,7 +145,7 @@ async def sequential_backward(
             try:
                 if attempt_no >= 1:
                     _, backup_inputs, backup_sequences = await sequential_forward(
-                        inputs, prompts, sequence_manager, start_index=span.start, end_index=span.end
+                        sequence_manager, inputs, prompts, start_index=span.start, end_index=span.end
```

Comment:
subjective matter: sequence_manager is the first parameter to most internal functions; can rollback if the reviewer disagrees.